### DISCUSSION TITLE
Imagick Support for 'use-sharp-yuv' on WebP

### DISCUSSION THREAD

**Discussion by jethrodesign** [ImageMagick/ImageMagick#3588] [Help]
Hi, we have a server running cPanel and CloudLinux 7.9. The CloudLinux version of PHP 7.4 includes the Imagick module compiled with support for WebP.

I see that ImageMagick can include the 'use-sharp-yuv' option when creating a WebP (https://imagemagick.org/script/webp.php). This is supposed to help with certain quality issues in regards to small, brightly-colored details.

But I can't find any information anywhere as to whether Imagick (which is using ImageMagick 7.0.10-10 Q16) has access to this option as well. 

Anybody have insight here?

**Comment by fmw42** [comment]
I doubt that the Imagick developer has included that yet. So there are not likely any hooks yet for it. Imagick is not maintained by Imagemagick. So you would need to contact the Imagick developer.

Why not just use PHP exec() with command line Imagemagick.

**Comment by urban-warrior** [comment]
`webp:use-sharp-yuv` is just a define.  We suspect Imagick supports defines although we have never tried it.
